src/style.js

```diff
@@ -21,15 +21,17 @@ export default class extends Component {
   }
 }
 
-export function flush() {
-  const ret = {}
-
-  for (const {props} of components) {
-    ret[props.styleId] = props.css
+function componentMap() {
+  const ret = new Map()
+  for (const c of components) {
+    ret.set(c.props.styleId, c)
```

Comment:
fair point (2) :) I'll wait a day to see if either @nkzawa or @rauchg have something to add and I'll merge. Thanks again!